### DISCUSSION TITLE
which block lists are you guys using

### DISCUSSION THREAD

**Discussion by suricloud** [mrrfv/cloudflare-gateway-pihole-scripts#10] [General]
.

**Comment by mrrfv** [comment]
I use the default ones, along with some personal Cloudflare Gateway firewall rules.

**Comment by suricloud** [comment]
along with the cgps list, if we create a new "adblock list" from cloudflare firewall policies and manually add domain to get blocked.
will this interfere with cgps ? is this good idea or not.

**Reply by mrrfv** [reply]
CGPS expects that you don't create any custom lists as that would interfere with the list item limit. However, you can create as much custom firewall policies as you want as long as the name isn't "CGPS Filter Lists".

**Reply by suricloud** [reply]
Ok,so better not to create custom blocklist?
Cause i taught to create a custom blocklist with another name and then add manually some domains to it.

**Reply by mrrfv** [reply]
Depends on whether you want to create the blocklist directly through Cloudflare or using CGPS.

**Reply by suricloud** [reply]
On cloudflare it self i would do.But i follow your recommendation.
So good or bad idea.

**Comment by suricloud** [comment]
Hi will this block list work ?

https://raw.githubusercontent.com/hagezi/dns-blocklists/main/hosts/pro.plus.txt

**Reply by mrrfv** [reply]
CGPS will most likely accept it, however it has 1,170,071 entries - that is way beyond the Cloudflare list item limit for the free plan of 300,000 domains. I suggest that you use a different list with less domains.

**Reply by vietthedev** [reply]
With the recent change, you can now use the wildcard domains format. But it will be truncated anyway because there are 326K entries, which exceeds the 300K limit. You can use the PRO variant or just keep using PRO++ that way and be aware that the ending 26K domains will be truncated.

**Comment by suricloud** [comment]
If i follow the step as you mentioned to create  private repositorie, which file should i then add more or edit to set my preferred filterlist and whitelist ? because for now i only have the main.yml file only ?
